test/alternator/conftest.py

```diff
@@ -72,7 +74,7 @@ def pytest_collection_modifyitems(config, items):
 # If this function can't connect to CQL, it will return an arbitrary
 # user/secret pair, and hope it would work if alternator-enforce-authorization
 # is off.
-@pytest.fixture(scope="session")
+@pytest.fixture(scope=testpy_test_fixture_scope)
```

Comment:
I really don't like this change, even if in practice it doesn't do anything because testpy_test_fixture_scope is also session when run through the "run" script (is it?). 
Basically, I don't understand why "session" is wrong. Just because in the old test.py happened to run each file in a separate processes and not share fixtures between files, does it mean we have to reproduce this wasteful behavior? The alternator test suite you're modifying here is designed to allow sharing these fixtures between files, so why not leave these fixtures "session" as designed?